renovate-changesets/dependencyConfig.ts

```diff
@@ -0,0 +1,10 @@
+import * as core from '@actions/core';
+
+export const getDependencyManager = (): string => {
+  return core
+    .getInput('dependency-manager', {
+      required: false,
+    })
+    .trim()
```

Comment:
FYI `core` already trims this for you. You have to pass an option if you don't want it to trim for you.